solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/Cell/RepositoryViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RepositoryViewCell.swift
+//  GitHubApp
+//
+//  Created by Felipe on 06/07/22.
+//
+
+import UIKit
+
+final class RepositoryViewCell: UITableViewCell {
+    
+    struct RepositoryViewCellConfiguration{
+        let repositoryName: String
+        let repsitoryOwnerName: String
+    }
+    
+    private lazy var containerVS: UIStackView = {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.spacing = 6
+        
+        return stack
+    }()
+    
+    private lazy var repositoryNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 18)
+        
+        return label
+    }()
+    
+    private lazy var repositoryOwnerNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .gray
+        label.font = UIFont.systemFont(ofSize: 14)
+        
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.accessoryType = .disclosureIndicator
+        setup()
+        
+        
+    }
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        return nil
+    }
+    
+    
+}
+extension RepositoryViewCell: ViewCode{
```

Comment:
Somente um nit de espaçamento aqui:
```suggestion
extension RepositoryViewCell: ViewCode {
```